libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -200,16 +200,20 @@ include AP_RangeFinder/AP_RangeFinder_Backend.h
 
 ap_object AP_RangeFinder_Backend depends (!defined(HAL_BUILD_AP_PERIPH) || defined(HAL_PERIPH_ENABLE_RANGEFINDER))
 ap_object AP_RangeFinder_Backend method distance float
+ap_object AP_RangeFinder_Backend method signal_quality_pct float
+ap_object AP_RangeFinder_Backend method signal_quality_pct rename signal_quality
 ap_object AP_RangeFinder_Backend method orientation Rotation'enum
 ap_object AP_RangeFinder_Backend method type uint8_t
 ap_object AP_RangeFinder_Backend method status uint8_t
 ap_object AP_RangeFinder_Backend method handle_script_msg boolean float'skip_check
+ap_object AP_RangeFinder_Backend method handle_script_msg2 boolean float'skip_check float'skip_check
```

Comment:
in the past we have used a structure to pass multiple things in one go. We could do it again here, but it would need to be a custom binding so we don't break scripts using the current method.

Reply:
@IamPete1 Thanks! I don't like polluting the lua interface either. If this method were implemented as a custom binding wouldn't it be possible to just make the second argument optional? I will investigate.

Reply:
@IamPete1 What do you think about this solution using an optional argument?